File: bannerlayout/src/main/java/com/wkz/bannerlayout/imagemanager/ImageLoaderManager.java
package com.wkz.bannerlayout.imagemanager;

import android.support.annotation.NonNull;
import android.view.ViewGroup;
import android.widget.ImageView;

import com.nostra13.universalimageloader.core.ImageLoader;
import com.wkz.bannerlayout.listener.BannerModelCallBack;
import com.wkz.bannerlayout.listener.ImageDisplayManager;

public class ImageLoaderManager implements ImageDisplayManager {
    @NonNull
    public ImageView display(@NonNull ViewGroup container, @NonNull BannerModelCallBack model) {
        ImageView imageView = new ImageView(container.getContext());
        ImageLoader imageLoader = ImageLoader.getInstance();
        imageLoader.displayImage((String) model.getBannerUrl(), imageView);
        return imageView;
    }
}

File: bannerlayout/src/main/java/com/fphoenixcorneae/bannerlayout/imagemanager/ImageLoaderManager.kt
package com.fphoenixcorneae.bannerlayout.imagemanager

import android.view.ViewGroup
import android.widget.ImageView
import com.fphoenixcorneae.bannerlayout.listener.BannerModelCallBack
import com.fphoenixcorneae.bannerlayout.listener.ImageDisplayManager
import com.nostra13.universalimageloader.core.ImageLoader

class ImageLoaderManager : ImageDisplayManager {
    override fun display(container: ViewGroup, model: BannerModelCallBack<*>?): ImageView {
        val imageView = ImageView(container.context)
        val imageLoader = ImageLoader.getInstance()
        imageLoader.displayImage(model?.bannerUrl as? String, imageView)
        return imageView
    }
}